src/ext-test/java/org/opentripplanner/ext/fares/impl/OrcaFareServiceTest.java

```diff
@@ -75,9 +75,9 @@ public static void setUpClass() {
    * types.
    */
   private static void calculateFare(List<Leg> legs, FareType fareType, Money expectedPrice) {
-    ItineraryFares fare = new ItineraryFares();
-    orcaFareService.populateFare(fare, USD, fareType, legs, null);
-    assertEquals(expectedPrice, fare.getFare(fareType));
+    var itinerary = new Itinerary(legs);
+    var itineraryFares = orcaFareService.calculateFares(itinerary);
+    assertEquals(expectedPrice, itineraryFares.getFare(fareType));
```

Comment:
In a future round of development we probably want to be more specific in our assertions and making sure that the fares are correctly assigned to the individual legs.

Reply:
There is a test that checks the leg information is correct, but you're right that eventually all the tests should be checking each leg.